src/parser.h calls two classes provided by dev-qt/qtbase[-xml] :
QXmlStreamReader
QXmlStreamAttributes
--- a/socnetv.pro
+++ b/socnetv.pro
@@ -16,7 +16,6 @@
 
 # add Qt module support
 QT += core
-QT += xml 
 QT += network
 QT += widgets
 QT += printsupport 
